### DISCUSSION TITLE
Emote Visibility

### DISCUSSION THREAD

**Discussion by Soha-lia** [KosmX/emotes#369] [Q&A] [answered]
Hi, 
After dl Emote mod, I have an issue. When I'm playing an emote, other friends cannot see it and vice versa. Idk what can I do to solve that ?

I'm French, sorry for my bad language :/

**Comment by AlexPug017** [comment]
hi, I encountered the same problem did you managed to fix it?

**Comment by treshevv** [comment]
u can fix that by adding emotectaft plugin/mod to your server. emotecraft is server based mod, u can't see othhe people emotes when u dont have a plugin or mod on ur server

**Comment by KosmX** [comment, marked as answer]
You have to install the mod both on server and clients.

If it still doesn't work, please send game logs.
